app/controllers/navigation/Navigator.scala

```diff
@@ -258,41 +262,67 @@ object Navigator {
     case TaricCode                 => controllers.declaration.routes.TaricCodeSummaryController.displayPage
     case TaricCodeFirst            => controllers.declaration.routes.CusCodeController.displayPage
     case StatisticalValue          => controllers.declaration.routes.NactCodeSummaryController.displayPage
-    case CommodityMeasure          => controllers.declaration.routes.PackageInformationController.displayPage
-    case DocumentsProduced         => controllers.declaration.routes.AdditionalInformationController.displayPage
   }
 
   val commonCacheDependent: PartialFunction[DeclarationPage, (ExportsDeclaration, Mode) => Call] = Map.empty
 
+  val commonCacheItemDependent: PartialFunction[DeclarationPage, (ExportsDeclaration, Mode, String) => Call] = {
```

Comment:
Nit: Those should be rather `commonItemCacheDependent`

Reply:
Depends how you read it in your head.  "common cache and item dependant" or "common item and cache dependant"?  Both make sense.

Reply:
Good point! I was just adding "ItemDependent" to the value names that already existed.